blog/themes/3mdeb/layouts/authors/terms.html

```diff
@@ -1,6 +1,37 @@
-<h1>Authors</h1>
-<ul>
-{{ range .Data.Pages }}
-  <li><a href="{{ .Permalink }}">{{ .Params.name }}</a></li>
-{{ end }}
-</ul>
+{{ partial "header.html" . }}
+{{ $pageAuthor := .Params.identifier }}
+
+<div class="container">
+  <div class="section">
+    <div class="columns">
+      <div class="column is-9">
+        <div class="tile is-child box">
+          <div class="content">
+            {{ range .Data.Pages }}
+              <h2 class="author-title"><a href="{{ .Permalink }}">{{ .Params.name }}</a></h2>
+              <img class="author-image" src="{{ .Params.photo }}" alt=""/>
+              {{ $quantity := 0 }}
```

Comment:
@artur-rs It count all Articles not per author

Reply:
`{{ $pageAuthor := .Params.identifier}}` was set in the wrong place (not at the top, but in the `range` function). I'm making changes right now